src/Symfony/Component/Form/Tests/Extension/Core/Type/ExtendedChoiceTypeTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\Extension\Core\Type;
+
+use Symfony\Component\Form\Forms;
+use Symfony\Component\Form\Tests\Fixtures\ChoiceTypeExtension;
+
+class ExtendedChoiceTypeTest extends \PHPUnit_Framework_TestCase
```

Comment:
the "underscore" is legacy in `PHPUnit_Framework_TestCase`, not the test case :)